dynamiq/nodes/tools/zenrows.py

```diff
@@ -9,20 +9,22 @@
 from dynamiq.utils.logger import logger
 
 
+class ZenRowsInputSchema(BaseModel):
+    url: str = Field(..., description="Parameter to provide a url of the page to scrape.")
```

Comment:
`url` is required by this. Is this expected? In such case there is no sense in ZenRowsTool.url

Reply:
Yes it is required as it is the only parameter. `url` can still be passed but be empty, so here behaviour is the same.

Reply:
I am not sure what is was used for to provide ZenRowsTool.url

Reply:
The URL can be a parameter from the input or from the node initialization.

Reply:
So this need to fix